src/constraints/GeomMeanEpiCone.jl

```diff
@@ -133,12 +65,13 @@ end
 
 function _add_constraint!(
     context::Context,
-    constraint::GeomMeanEpiConeConstraint,
+    constraint::GenericConstraint{GeomMeanEpiCone},
 )
-    A = constraint.cone.A
-    B = constraint.cone.B
-    t = constraint.cone.t
-    T = constraint.T
+    n = constraint.set.side_dimension
+    T = constraint.child[1:n^2]
+    A = constraint.child[n^2 .+ (1:n^2)]
+    B = constraint.child[2n^2 .+ (1:n^2)]
```

Comment:
indexing in Convex is non-trivial (it's another sparse affine operation); I'd worry that this could regress problem formulation (and possibly even solve times). In Convex's current setup, I'd wonder if adding a non-concretely typed field to GenericConstraint to carry extra data would be better, and that maybe a dynamic dispatch is better than indexing for Convex.

Reply:
Maybe we could allow `child` to be a `Tuple` and interpret it as a `vcat`

Reply:
yeah, if it's a `Tuple` with elements of `AbstractExpr`, then indexing is just normal julia indexing, so there should be no issues there.